app/modules/account.js

```diff
@@ -18,8 +19,15 @@ export function logout(){
   }
 };
 
+export function showKey(isToggled) {
```

Comment:
This toggling state should be handled by the Login component itself for coherence. This state change isn't application wide necessary.